core/src/entities/mod.rs

```diff
@@ -21,3 +21,6 @@ pub mod xrefs;
 
 /// Persistence and domain models that support enhancing SBOMs with additional metadata.
 pub mod enrichments;
+
+/// Models for Analytics
```

Comment:
Yes!  You are correct.  We can also just blow them away because I'm not using them.  I'm closing this conversation because this context has been addressed in another conversation.